cmd/orders/list_open.go

```diff
@@ -65,10 +65,10 @@ var listOpenOrdersCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(listOpenOrdersCmd)
+	Cmd.AddCommand(listOpenOrdersCmd)
 
-	listOpenOrdersCmd.Flags().StringP(utils.ProductIdFlag, "i", "", "ID of the product")
-	listOpenOrdersCmd.Flags().StringP(utils.PortfolioIdFlag, "", "", "Portfolio ID. Uses environment variable if blank")
+	listOpenOrdersCmd.Flags().String(utils.ProductIdFlag, "", "ID of the product")
```

Comment:
this print out includes the request object
`❯ ./prime orders list-open --product-id ETH-USDC | jq
{
  "orders": [],
  "request": {
    "portfolio_id": "d7a7abc5-...-9252a740a3c8",
    "product_id": "ETH-USDC"
  }
}`